# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/moz.build
# Commit: bb094a6084b3
# Full Hash: bb094a6084b3b127c090827ee8ede4b06cdbdea6
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2020-03-19 17:04:52
# Regressor Bug: 1623278
# File Overlap Count: 1
# Description:
#   Bug 1623278 - Use SafeRefPtr for FileManager. r=dom-workers-and-storage-reviewers,asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D65549
# ==============================================================================

diff -r 7fdf38a403bb -r bb094a6084b3 dom/indexedDB/moz.build
--- a/dom/indexedDB/moz.build	Thu Mar 19 09:32:02 2020 +0000
+++ b/dom/indexedDB/moz.build	Thu Mar 19 09:32:53 2020 +0000
@@ -42,6 +42,7 @@
     'IndexedDatabase.h',
     'IndexedDatabaseManager.h',
     'InitializedOnce.h',
+    'SafeRefPtr.h',
 ]
 
 EXPORTS.mozilla.dom.indexedDB += [